ansible/ncn_nodes.yml

```diff
@@ -65,6 +65,46 @@
       when: ansible_distribution_file_variety == "SUSE"
     - role: csm.ncn.sat
 
+# Install SBPS iscsi target packages on iscsi targets(worker nodes)
+- hosts: Iscsi_Targets
```

Comment:
The comment on the previous line makes it sound like this is supposed to run on all worker NCNs. Is that true? If so, this line (and the other `hosts:` lines) should be changed accordingly. I'm assuming you've just got it this way to help you in testing it (although you could accomplish the same thing just by creating CFS sessions limited to the worker nodes you want it to run on)

Reply:
yes, this is going to be with CFS session as you suggested. Thanks.

Reply:
Ok, so should the`hosts` field be `Iscsi_Targets` or `Management_Worker` ?
If this is supposed to run on all NCN workers and nowhere else, then shouldn't we use the latter?

Reply:
@mharding-hpe, Ravi and I discussed last week. By default in CSM 1.6, the roles should not be run unless an administrator modifies the playbook content to enable. When enabling, the administrator can choose to retain teh default set of nodes (e.g., Management_Worker), or restrict it to a subset of worker nodes.